static/directives/quota-management-view.html

```diff
@@ -1,48 +1,52 @@
 <div class="quota-management-view-element">
+  <div ng-if="view == 'organization-view' && !showConfigPanel && !prevquotaEnabled">
+    <button class="margin-2 add-quota-limit btn btn-success hidden-xs" ng-click="toggleOrgConfigView()">
+      <i class="fa margin-right-4"></i>
+      Add Storage Quota
+    </button>
+  </div>
+  <div ng-if="!(view == 'organization-view' && !showConfigPanel && !prevquotaEnabled) || view == 'super-user'">
+  <fieldset ng-disabled="view == 'organization-view' && prevquotaEnabled">
   <form>
   <table class="co-list-table">
     <tr>
-      <td>
-        Quota Management:
+      <td class="row-alignment">
+        Set storage quota:
       </td>
       <td>
 	<table class="co-table">
 	  <td>
-            <input class="margin-2 form-control width-440" type="number" name="quota-limit" min="1" ng-model="currentQuotaConfig['quota']">
+            <input class="margin-2 form-control width-440" type="number" name="quota-limit" ng-model="currentQuotaConfig['quota']">
```

Comment:
Because adding a check [here](https://github.com/quay/quay/pull/1242/files#diff-97350d41b191619358d2e04565d8f58061faafcf99224960fe1cd38bc9d79846R323). If I let `min=1` be, it returns undefined and I cannot validate against it to throw appropriate error msg.